command/token_renew.go

```diff
@@ -77,8 +77,8 @@ func (c *TokenRenewCommand) Flags() *FlagSets {
 		Default:    0,
 		EnvVar:     "",
 		Completion: complete.PredictAnything,
-		Usage: "Request a specific increment for renewal. Vault is not required " +
-			"to honor this request. If not supplied, Vault will use the default " +
+		Usage: "Request a specific increment for renewal. Vault will not honor this" +
+			"request for periodic tokens. If not supplied, Vault will use the default " +
```

Comment:
I have no idea about this, but if we think there might be cases other than periodic tokens where increment isn't honored, we may want to leave this more open and hybridize the statement between "is not required to honor" and restricting the limit to periodic tokens?

Reply:
Like if there are global maximums that would restrict how long the token could be renewed for?

Reply:
Aha yes that is absolutely correct, just noticed this after you mentioned it https://github.com/hashicorp/vault/blob/main/sdk/framework/lease.go#L100, I'll wordsmith this a bit more, thanks for the catch!

Reply:
@mladlow how does "This increment may not be honored, for instance in the case of periodic tokens." sound ?

Reply:
Sounds good to me!

Reply:
1c8dad2